examples/pytorch/igmc/.gitignore

```diff
@@ -0,0 +1,5 @@
+__pycache__
+.vscode/
+log/
+*.ipynb
+raw_data/
```

Comment:
If https://github.com/dmlc/dgl/blob/master/.gitignore is not sufficient, you should edit that file rather than create a new file.